### DISCUSSION TITLE
How do I hide the Minimize | Configure in the upper left corner of the album play screen?

### DISCUSSION THREAD

**Discussion by kevinclevenger** [jeffvli/feishin#766] [General]
I really dislike the corner menu, that screen already has way too much padding between elements and this makes it's worse.  

How do I hide it? Is there something I can add to a custom .css? What version did it start in?  

![Screenshot from 2024-09-28 07-05-22](https://github.com/user-attachments/assets/7c74f2a4-3ec6-4ed3-bc51-a4b061d9f6ca)

**Comment by jeffvli** [comment]
I don't remember what version it was added.

The background can be made transparent if you set the opacity to 0 in the config popup. 

![image](https://github.com/user-attachments/assets/3752034d-7882-446d-a440-5cb3f4eca65d)

There is also a `custom css` option under the `Advanced Tab` in the settings if you want to modify it further.

**Comment by kevinclevenger** [comment]
Thanks for responding. To be clear, I'm after removing/hiding the upper left minimize/settings.  

| There is also a custom css option under the Advanced Tab in the settings if you want to modify it further.  

Yes but what is the css element, class, etc? What should the css for that look like.

![Screenshot from 2024-09-29 08-28-31](https://github.com/user-attachments/assets/f0a6f359-be8e-482b-87bd-49a4ed7bb34b)

**Reply by jeffvli** [reply]
Rough solution, but this should work:

```css
#main-content > div:first-child > div:first-child {
       display: none;
}
```

![image](https://github.com/user-attachments/assets/cea9d8ed-8eae-44aa-9f45-30dbe6629446)

**Comment by kevinclevenger** [comment]
That works great. Thank you.  

Is there any guidance on minimizing the vertical space between the top of the album art/track list and the bottom of the title bar?
